website1/src/scss/sections/_footer.scss

```
@@ -0,0 +1,62 @@
+.footer {
+  &-wrapper {
+    width: 100%;
+    background-color: $white-back;
+
+  }
+}
+
+.footer-content {
```

Move to footer class
```suggestion
&-content {
```